website/header_menu_items.py

```diff
@@ -37,6 +37,10 @@ class HeaderLink:
             HeaderLink(label=_("Our Sponsors"), href="/sponsors"),
         ]
     ),
+    HeaderLink(
+        label=_("Grant"),
```

Comment:
```suggestion
        label=_("Opportunity Grants"),
```
Can we change this to `Opportunity Grants` so its clear what the link is for?